app/services/resource_service.py

```diff
@@ -423,3 +423,18 @@ async def fetch_contact_info(request_body: object, params: Optional[Dict]) -> Di
     # TODO do error-handling over here
 
     return response
+
+
+@retry(attempts=3, delay=1, retry_exceptions=(TooManyRequestException,))
```

Comment:
why extra comma in the end?
@retry(attempts=3, delay=1, retry_exceptions=(TooManyRequestException,))

Reply:
That's actually the convention.
This syntax is specifically used to define a tuple with a single element.